exit.go

```diff
@@ -549,6 +549,25 @@ func (te *TunaExit) StartReverse(shouldReconnect bool) error {
 			continue
 		}
 
+		reverseIP := tcpConn.RemoteAddr().(*net.TCPAddr).IP
+		reverseTCP := reverseMetadata.ServiceTcp
+		if len(reverseTCP) > 0 {
+			go func() {
+				net.DialTimeout(tcp, fmt.Sprintf("%s:%d", reverseIP.String(), reverseTCP[0]), defaultReverseTestTimeout)
+			}()
+
+			session.SetDeadline(time.Now().Add(defaultReverseTestTimeout))
```

Comment:
Could we call `SetDeadline` when session init?

Reply:
This is to prevent `session.Accept()` not returning (if entry port is not open)